lib/cdo/contact_rollups.rb

```diff
@@ -527,7 +515,10 @@ def self.update_district
     users = User.where("length(email) > 0")
 
     users.find_each do |user|
-      PEGASUS_REPORTING_DB_WRITER[DEST_TABLE_NAME.to_sym].where(email: user.email).update(school_name: user.ops_school) unless user.ops_school.nil?
+      unless user.ops_school.nil?
+        PEGASUS_REPORTING_DB_WRITER[DEST_TABLE_NAME.to_sym].where(email: user.email).
+            update(school_name: user.ops_school)
+      end
```

Comment:
Any reason why you split up this conditional into multiple lines? FWIW our [styleguide](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier) prefers the modifier usage when it's a single line body.